cmd/agent/checkers.go

```diff
@@ -68,6 +68,7 @@ func addToMaster(node agent.Agent, config *config, kubeConfig monitoring.KubeCon
 	node.AddChecker(monitoring.DockerHealth(config.dockerAddr))
 	node.AddChecker(etcdChecker)
 	node.AddChecker(monitoring.SystemdHealth())
+	node.AddChecker(monitoring.PingHealth(config.serfRPCAddr,config.role)
```

Comment:
nit: missing white-space before the comma - `go fmt` should've fixed that.

Reply:
fixed and activated `gofmt` onSave (on my current editor)